.travis.yml

```diff
@@ -43,6 +43,7 @@ env:
   - PACKAGE=surelog
   - PACKAGE=sv-parser
   - PACKAGE=verible             USE_SYSTEM_GCC_VERSION="8"
+  - PACKAGE=uhdm-integration
```

Comment:
This should probably be a couple of separate packages;
 * `yosys-udhm`
 * `verilator-uhdm`
 * `surelog-parse`

Reply:
@mithro we actually generate two packages from this (there are two outputs defined for `uhdm-yosys` and `uhdm-verilator`) - it probably doesn't make sense to split that in travis as a major part for both outputs is common (namely `surelog` + `uhdm`).

Both packages come with `surelog` inside as there are several reasons to keep it this way:
* The development is quite rapid, having them packed together allows us to be sure that the versions are compatible (`surelog` used here is actually vanilla `surelog` but from a submodule that should be fully compatible with the rest of the integration)
* The way `sv-tests` travis jobs are currently implemented is they uncomment a dependency and download a single conda package depending on the name of the job (I think you are the author of this mechanism) - if we pack `surelog` with `yosys`/`verilator` we won't have to change that mechanism - we probably could try to work around this by defining some dependencies in conda, but the way it is done now just makes things simpler
* In the long run we may want to consider using `surelog` and `udhm` as libraries and compile them into `verilator`/`yosys` to avoid that extra step of dumping `udhm` output into file